src/main/java/com/sourcery/gymapp/backend/workout/mapper/RoutineMapper.java

```diff
@@ -0,0 +1,30 @@
+package com.sourcery.gymapp.backend.workout.mapper;
+
+import com.sourcery.gymapp.backend.workout.dto.CreateRoutineDto;
+import com.sourcery.gymapp.backend.workout.dto.ResponseRoutineDto;
+import com.sourcery.gymapp.backend.workout.model.Routine;
+import org.springframework.stereotype.Component;
+
+import java.util.UUID;
+
+@Component
+public class RoutineMapper {
+    public ResponseRoutineDto toDto(Routine routine) {
```

Comment:
minor: isn't it better to name it entity? So you briefly understand it comes from DB?
so, public ResponseRoutineDto toDto(Routine entity
(not sure about this comment though)

Reply:
We can do that for clarity, but should this be then routineEntity or just routine?

Reply:
imo just (Routine entity)